src/__tests__/option.features.js

```diff
@@ -41,6 +41,13 @@ const testFeature = function(
   // enable only the one we want to test...
   options.features[feature] = true
 
+  if (featureDep[feature]) {
```

Comment:
Because the `nesting` feature is using `custom-selector` we need to enable `customSelectors` feature as well.